src/assets/server/rest/v1/schemas/transaction/transactions-get.json

```diff
@@ -0,0 +1,87 @@
+{
+  "title": "Get Transactions Request",
+  "type": "object",
+  "properties": {
+    "ChargeBoxID": {
+      "$ref": "chargingstation.json#/definitions/id"
+    },
+    "Issuer": {
+      "type": "boolean",
+      "sanitize": "mongo"
+    },
+    "ConnectorID": {
+      "$ref": "chargingstation.json#/definitions/connector/properties/connectorId"
+    },
```

Comment:
Like this it is only possible to filter for one connector. If you filter for multiple connectors you will get an error in the front end.

Reply:
Fixed